src/components/dashboard/SearchList.jsx

```diff
@@ -0,0 +1,70 @@
+// Copyright 2019 Stanford University see LICENSE for license
+
+import React, { useState, useEffect } from 'react'
+import PropTypes from 'prop-types'
+import { useSelector } from 'react-redux'
+import SearchRow from './SearchRow'
+import useSearch from 'hooks/useSearch'
+import { useHistory } from 'react-router-dom'
+import { selectSearchQuery } from 'selectors/search'
+
+
+/**
+ * This is the list view of searches
+ */
+const SearchList = (props) => {
+  const { fetchNewSearchResults } = useSearch()
+  const [navigateSearch, setNavigateSearch] = useState(false)
+  const query = useSelector((state) => selectSearchQuery(state, 'resource'))
+  const history = useHistory()
+
+  // Need to wait until results before navigating
+  useEffect(() => {
+    if (navigateSearch && query) history.push('/search')
+  }, [navigateSearch, query, history])
+
+  const handleSearch = (queryString, uri, event) => {
+    event.preventDefault()
+    fetchNewSearchResults(queryString, uri)
+    setNavigateSearch(true)
+  }
+
+  const rows = props.searches.map((row) => (
+    <SearchRow
+        row={row}
+        key={`${row.authorityUri}-${row.query}`}
+        handleSearch={handleSearch}
+    />
+  ))
+
+  return (
+    <div className="row">
+      <div className="col">
+        <table className="table table-bordered search-list">
+          <thead>
+            <tr>
+              <th style={{ backgroundColor: '#F8F6EF', width: '30%' }}>
+                Authority
+              </th>
+              <th style={{ backgroundColor: '#F8F6EF', width: '66%' }}>
+                Query
+              </th>
+              <th style={{ backgroundColor: '#F8F6EF', width: '4%' }}
```

Comment:
Since the `backgroundColor` is common to all three of these table headers, maybe move them to a CSS class?